### DISCUSSION TITLE
How to disable more videos on yuotube

### DISCUSSION THREAD

**Discussion by ghost** [videojs/video.js#7977] [General]
How to hide more videos in embed youtube videos in videos js the tag rel=0 was disabled by youtube api

Thanks

**Comment by mister-ben** [comment]
The YouTube tech can only set parameters the YouTube embed supports. The only thing I can think of is whether setting the current time to 0 on `ended`, perhaps with a quick play and pause, might clear it.

**Reply by ghost** [reply]
I tryed but keep brings from the channel of. the video, a friend mine give a idea to use nodejs as server and sync frame from youtube of the video and block all others contents embed in url.

Thanks for reply
